dashboard/settings.py

```diff
@@ -338,43 +338,43 @@
     LOGIN_REDIRECT_URL = '/'
 
 # Give an opportunity to disable LTI
-if config('STUDENT_DASHBOARD_LTI', default='False', cast=bool):
+if ENV.get('STUDENT_DASHBOARD_LTI', False):
     INSTALLED_APPS += ('django_lti_auth',)
 
-    PYLTI_CONFIG = {
-        "consumers": json.loads(config("PYLTI_CONFIG_CONSUMERS", default="{}", cast=str)),
+    PYLTI_ENV.get = {
```

Comment:
My build is failing at this line
```
File "/code/dashboard/settings.py", line 345, in <module>
    "consumers": json.loads(ENV.get("PYLTI_ENV.get_CONSUMERS", {})),
  File "/usr/local/lib/python3.6/json/__init__.py", line 348, in loads
    'not {!r}'.format(s.__class__.__name__))
TypeError: the JSON object must be str, bytes or bytearray, not 'dict'
```

Reply:
pylint is showing an error on that line
![Screen Shot 2019-06-26 at 8 22 32 AM](https://user-images.githubusercontent.com/8579775/60179549-ecf37e00-97eb-11e9-8c80-0586cba4d1c2.png)

Reply:
Okay latest update should fix this. Thanks!